### PR TITLE
Added the fifth chapter of Nest gs

### PR DESCRIPTION
**Pull request by Pedrochu**

### CODE REVIEW

File: nest/05.cheat_sheet.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# 5 Cheat sheet
+In this chapter you will find a cheat sheet, detailing all the commands and options available for `nest` and `finest`, with a short explanation on what they do, and how to use them.
```
Comment (Gaspard--) [inline nest/05.cheat_sheet.md]:
```suggestion
In this chapter you will find a cheat sheet, detailing all the commands and options available for `nest` and `finest`, with a brief explanation on what they do, and how to use them.
```
(no difference in meaning, I just prefer brief)

**Review by Gaspard-- — COMMENTED** [review]
Stupid idea, but maybe you could say what \<package>, \<req>, \<group> and \<repository> are and link to the relevant chapters  in a section before `##nest` ? I don't think it's necessary, but if you find some quick condensed format it would be a cool addition.

File: nest/05.cheat_sheet.md (hunk 1 above)
Comment (Pedrochu) [reply nest/05.cheat_sheet.md]: Could you rephrase the first sentence of your comment? :grimacing:

Comment (Gaspard--) [reply nest/05.cheat_sheet.md]: I meant that I prefer "brief" over "short", but that there is no difference in meaning between both versions.

Comment (Pedrochu) [reply nest/05.cheat_sheet.md]: I meant from your other comment. The one just above starting with `Stupid idea`, it's like words are missing or something

Comment (Gaspard--) [reply nest/05.cheat_sheet.md]: damn, apparently markdown swallows words in <>
Going to edit that.
(done)

File: nest/04.finest.md [hunk 2]
```diff
@@ -174,7 +175,7 @@ Removing group @WorkGroup1...
 Successfully removed all the specified groups.
 ```
 
-#### List groups
+#### List the groups
```
Comment (Arignir) [inline nest/04.finest.md]:
```suggestion
#### List all groups
```

File: nest/05.cheat_sheet.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* `nest install <package>` installs a package.
+* `nest uninstall <package>` uninstalls a package
+* `nest upgrade <package>` performs a full upgrade of the system, or upgrades the packages provided.
+* `nest list` lists all the installed packages
+
+## `finest`
+* `finest requirement add <req> [--parent <parent>]` adds a requirement to the given parent group (default: `@root`)
+* `finest requirement remove <req> [--parent <parent>]` removes a requirement from the given parent group (default: `@root`)
+* `finest group create <group> [--parent <parent>]` creates a group to the given parent (default: `@root`)
+* `finest group delete <group>` removes a group if the group is not empty, use `-f` or `--force`flag to force the removal
```
Comment (Arignir) [inline nest/05.cheat_sheet.md]:
```suggestion
* `finest group delete <group>` removes a group if the group is not empty, use the flag `-f` or `--force` to force the removal
```

File: nest/05.cheat_sheet.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* `finest group list` lists all existing groups on the OS
+* `finest pull <repository>` pulls all of the repositories if none are given
+* `finest merge` this command must be executed after transactions were executed in order to merge them in your current dependency graph
```
Comment (Arignir) [inline nest/05.cheat_sheet.md]: This decription doesn't follow the same format than the previous one, which is odd.

**Review by Arignir — CHANGES_REQUESTED** [review]
